truthy-falsy.js

```diff
@@ -0,0 +1,206 @@
+// 1
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(true, false));
+
+// 2
+function closeCompare(firstNumber, secondNumber, numbersMargin = 0){
+  const numbersDiff = Math.abs(firstNumber - secondNumber);
+  if (numbersDiff <= numbersMargin) {
+    return 0;
+  }
+  if (firstNumber < secondNumber) {
+    return -1;
+  }
+  return 1;
+}
+
+// 3
+function or(firstVariable, secondVariable) {
```

Comment:
`firstVariable` is a little too generic. We know that it is a boolean, so we could call it the `firstBoolean`